frontend/src/App.js

```diff
@@ -106,8 +106,8 @@ function App() {
           <Route exact path="/logout">
             <LogoutSuccess />
           </Route>
-          <Route exact path="/reviewSuccess">
-            <ReviewSuccess />
+          <Route exact path="/reviewSuccess/:id">
+            <ReviewSuccess hikeid={window.location.pathname.split('/')[2]} />
```

Comment:
Does this line basically take the url, split it by the forward slashes and get the second element to get the hike id?